public/teach-course.html

```
@@ -0,0 +1,163 @@
+              <button type="button" data-action="quote" title="Quote">❝</button>
+              <button type="button" data-action="ul" title="Bulleted List">≣</button>
+              <button type="button" data-action="ol" title="Numbered List">☰</button>
+              <button type="button" data-action="link" title="Insert Link">🔗</button>
+              <button type="button" data-action="separator" title="Separator">◉</button>
+              <button type="button" data-action="help" title="Formatting Help">?</button>
+            </div>
+            <textarea id="description" placeholder="Describe your course..."></textarea>
+          </div>
+          <div class="hint">Provide a detailed description of your course</div>
+        </div>
+        <input id="subject" type="text" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)